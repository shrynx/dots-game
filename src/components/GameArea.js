// @flow
import styled from 'styled-components';
import { BACKGROUND_COLOR } from '../styles/colors';

const GameArea = styled.div`
  height: 75vh;
  display: flex;
  align-items: center;
  background-color: ${BACKGROUND_COLOR};
  position: relative;
  overflow: hidden;
`;

export default GameArea;
