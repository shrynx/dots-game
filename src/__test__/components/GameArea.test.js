import React from 'react';
import { render } from 'react-testing-library';
import GameArea from '../../components/GameArea';

it('GameArea component should render correctly', () => {
  const { container } = render(<GameArea />);
  expect(container).toMatchSnapshot();
});
