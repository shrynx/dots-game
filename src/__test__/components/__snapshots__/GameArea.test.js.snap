// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameArea component should render correctly 1`] = `
<div>
  <div
    class="sc-bdVaJa eLEbYQ"
  />
</div>
`;
